src/Button.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './Button.css';
+
+function Button(props) {
+    return (
+        <button className={`button ${props.modifierClassName}`} {...props}>
+            {props.text}
+        </button>
```

Comment:
One thing I would consider here is just letting react's build in `children` prop do its magic instead of relying on a custom `text` prop. The react `children` prop works like this:

```jsx
<button>Text</button>
```

is equivalent to
```jsx
<button children="Text" />
```

So if, instead of asking for a custom `text` prop you just pass down all the props as is, you would naturally get this working for free:

```jsx
<Button className="continue-button">Text</Button>
```

(Of course this raises questions about why we have our own button component at all if we're just going to keep passing props wholesale down to its only child, but I think this proves useful in that you can now guarantee that every button has a class `button` and I imagine you will find more reusable standard features you want in the future.)